src/Covid19Dashboard/ControlPanel/index.jsx

```diff
@@ -11,9 +11,10 @@ import './ControlPanel.less';
 
 class ControlPanel extends PureComponent {
   render() {
-    const tooltipText1 = 'Google Mobility Data shows movement trends as a percentage variance from an established baseline for a given day and place.';
+    const tooltipText1 = (<React.Fragment> Google Mobility Data shows movement trends as a percentage variance from an established baseline for a given day and place. <br /> Google LLC &quot;Google COVID-19 Community Mobility Reports&quot;. <a href='https://www.google.com/covid19/mobility/'> https://www.google.com/covid19/mobility/</a>  <br /> Accessed: Today</React.Fragment>);
```

Comment:
i think we might need to remove the spaces before and after the `<br />` tags?

Reply:
I don't think the spaces matter. The image in the slack thread looks right to me. ([here](https://slack-files.com/T03A08KRA-F03KLGZBW5D-8256f2ee86)) Let me know if you still want me to double check.